indico/modules/search/result_schemas.py

```diff
@@ -199,6 +199,29 @@ class EventNoteResultSchema(ResultSchemaBase):
     event_path = fields.Method('_get_event_path', dump_only=True)
 
     def _get_url(self, data):
+        note_id = data['note_id']
+        event_id = data['event_id']
+        note = EventNote.get(note_id)
+        event = Event.get(data['event_id'])
+
+        if not event:
```

Comment:
`or event.type_ == EventType.conference` - the anchors don't work well for those events.

Reply:
Thanks, fixed :)